public/views/index/body.ejs

```diff
@@ -12,7 +12,8 @@
                         <div class="fr-header__service lasuite-header__service">
                             <a class="lasuite-header__service-link ui-home" href="/" title="Accueil - Notepad de l'État" aria-label="Accueil - Notepad de l'État">
                                 <img src="/build/lasuite/logo.svg" alt="" class="lasuite-header__service-logo fr-responsive-img" width="32" height="32">
-                                <p class="fr-header__service-title lasuite-header__service-title">Notepad de l'État</p>
+                                <p class="fr-header__service-title lasuite-header__service-title beta-tag">Notepad de l'État</p>
+                                <img src="<%- serverURL %>/icons/beta.svg" alt="" style="margin-bottom: 15px; margin-left: 5px;"  width="40">
```

Comment:
Idk where to declare styles in this project … Are inline styles acceptable?

Reply:
For the homepage/history page, it's the "cover" page, you can add css in `cover.css` if you prefer.